cli/lib/types/wordpress-server-ipc.ts

```
@@ -39,6 +44,7 @@ const managerMessageRunBlueprint = z.object( {
 
 const managerMessageStopServer = z.object( {
 	topic: z.literal( 'stop-server' ),
+	data: z.object( {} ),
```

pm2 requires all messages to have `id`, `topic`, and `data` fields. We didn't notice that the `stop-server` topic would generate errors because `stopWordPressServer` swallows exceptions.